tlslite/keyexchange.py

```diff
@@ -734,3 +718,88 @@ def makeClientKeyExchange(self):
         cke = super(SRPKeyExchange, self).makeClientKeyExchange()
         cke.createSRP(self.A)
         return cke
+
+
+class RawDHKeyExchange(object):
+    """
+    Abstract class for performing Diffe-Hellman key exchange.
+
+    Provides a shared API for X25519, ECDHE and FFDHE key exchange.
+    """
+
+    def __init__(self, group, version):
+        """
+        Set the parameters of the key exchange
+
+        Sets group on which the KEX will take part and protocol version used.
+        """
+        self.group = group
+        self.version = version
```

Comment:
Is ```self.version``` used anywhere?

Reply:
not now, but it will be used later when the changes necessary for TLS 1.3 are implemented